src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,98 @@
+.card {
+  width: $card-width;
+  margin-right: $card-margin-right;

```

Comment:
check it . I suggest to create all-products__card element and move margin there
![image](https://user-images.githubusercontent.com/122989585/224394390-cb1134e9-2733-4ae0-a448-c1b503836e96.png)